code/datums/wires/syndicatebomb.dm

```diff
@@ -16,11 +16,18 @@
 
 /datum/wires/syndicatebomb/on_pulse(wire)
 	var/obj/machinery/syndicatebomb/B = holder
+	var/mob/user
+	if(usr)
+		user = usr
```

Comment:
```suggestion
	var/mob/user = usr
	if(user)
```

Reply:
is usr is null, then the cast user will inherit that... makes sense